strings_lab.js

```diff
@@ -0,0 +1,66 @@
+// let str = "johanne";
```

Comment:
One common thing that I see from fellows' assignments is hard coding where code is working for one specific input but does not work as soon as I put different input. Always try to write code that can accommodate differing input and still gives us consistent output as a result. Keep up the good work and keep practicing!